list.h

```diff
@@ -0,0 +1,27 @@
+#ifndef LIST_H_
+#define LIST_H_
+
+enum
+{
+	ERR_NO_NODE_TO_POP = -1,
+	ERR_NO_VALUE_TO_POP = -3
+};
+
+typedef struct nodeS
+{
+	struct nodeS* next_p;
+	int value;
+} nodeS;
+
+typedef struct listS
+{
+	nodeS* head_p;
+} listS;
+
+int push_back(listS* list, int value);
+int pop_first(listS* list, int *valueBuff);
+int pop_last(listS* list, int *valueBuff);
```

Comment:
Okay, you wrote pop_first, pop_last but you've got only push_back. I'd be nice to have push/pop - front/back.